app/views/second_level_browse_page/_links.html.erb

```diff
@@ -1,6 +1,14 @@
 <div class="browse__inner <%= page.lists.curated? ? 'browse__inner--curated' : 'browse__inner--alphabetical' %>">
   <h2 tabindex="-1" class="browse__heading js-heading"><%= t("shared.browse.prefix") %> <%= page.title %></h2>
 
+  <%= render "govuk_publishing_components/components/intervention", {
+    suggestion_text: "Help improve GOV.UK",
+    suggestion_link_text: "Take part in user research",
+    suggestion_link_url: "https://gdsuserresearch.optimalworkshop.com/treejack/35gap0ij",
```

Comment:
This needs to be updated to https://GDSUserResearch.optimalworkshop.com/treejack/lb5eu75l